src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -4,11 +4,7 @@
         {% if forloop.last == true %}
             <li><a aria-current="page" href="/{{ path }}">{{ crumb.text }}</a></li>
         {% elsif crumb.url.path %}
-            {% if crumb.url.path == '/' %}
-                <li>
-                    <a href="{{ crumb.url.path }}" onClick="recordEvent({ event: 'nav-breadcrumb', 'nav-breadcrumb-section': 'home' });">{{ crumb.text }}</a>
-                </li>
-            {% else %}
+            {% if crumb.url.path != '/' %}
                 <li><a href="{{ crumb.url.path }}">{{ crumb.text }}</a></li>
             {% endif %}
```

Comment:
Huh....Is there a story behind this? I don't see much in the issue. 

This is okay, but I'm curious on the LOE for just removing the index page from `entityUrl.breadcrumb` in Drupal, so we can just render without special cases.

Reply:
So this ticket was brought up out of a meeting with @jenniferlee-dsva and @rtwell! They will likely have more info into the reasoning why the `Home` breadcrumb should be removed! :)

Reply:
Oh! Yes, if we can do this on Drupal I would much prefer that so the FE can be dumber! 👍

Reply:
Who would we reach out to for that ask @ncksllvn?

Reply:
I followed up in the issue to the CMS folks